app/models/user.rb

```diff
@@ -188,7 +191,7 @@ def not_a_third_party?
   end
 
   def with_invalid_email?
-    roles.include?('guest') || roles.include?('preview_session')
+    roles.include?('guest') || roles.include?('preview_session') || roles.include?('predefined_participant')
```

Comment:
I wonder, can we use `(roles & ['guest', 'preview_session', 'prefefined_participant']).any?` to check if `roles` overlap with the roles provided 🤔

Reply:
Or some other way to check if there's a common role between these two